native/src/main/java/io/ballerina/stdlib/grpc/protobuf/utils/BalFileGenerationUtils.java

```diff
@@ -64,6 +67,23 @@ public static void generateDescriptor(String command) throws CodeGeneratorExcept
             throw new CodeGeneratorException("Error in executing protoc command '" + command + "'. " + e.getMessage(),
                     e);
         }
+        try {
+            reader = new BufferedReader(new InputStreamReader(process.getErrorStream()));
+            String line;
+            while (true) {
+                try {
+                    if ((line = reader.readLine()) == null) {
+                        break;
+                    }
+                    output.add(line);
+                } catch (IOException e) {
+                    throw new CodeGeneratorException("Failed to read protoc command output. " + e.getMessage(), e);
+                }
+            }
+            reader.close();
```

Comment:
Close the reader inside finally block. or use try-with-resource

Reply:
https://docs.oracle.com/javase/tutorial/essential/exceptions/tryResourceClose.html